partnerships/roles/community-success-manager.md

```diff
@@ -0,0 +1,7 @@
+```{role} Community Success Manager
+```
+# Community success manager
+
+```{warning} To add
+To add here.
```

Comment:
cc @colliand and @jmunroe you'll need to define this role more clearly. If it's not ready yet that's OK, we can still merge, but please make sure this gets done relatively soon.